tests/test-cases/log-level-validation.yaml

```diff
@@ -34,8 +32,6 @@ tests:
     expect:
       diff: []
       exit_code: 1
-      stderr:
-        - "Error: Invalid log level 'XTrace'. Valid options are: \\[Trace Debug Info Warning Off\\]"
```

Comment:
Why did we remove this?

Reply:
We already have golden snapshot for this test. Having multiple assertions that do the same feels redundant.

Reply:
done